File: app/src/main/java/com/github/kongpf8848/animation/activity/viewpager/HookLayoutCallback.java
package com.github.kongpf8848.animation.activity.viewpager;

import android.content.Context;
import android.util.AttributeSet;
import android.view.View;

public interface HookLayoutCallback {
    void onCreateViewCallback(View parent, String name, Context context, AttributeSet attrs, View destView);
}

File: app/src/main/java/com/github/kongpf8848/animation/activity/viewpager/HookLayoutCallback.kt
package com.github.kongpf8848.animation.activity.viewpager

import android.content.Context
import android.util.AttributeSet
import android.view.View

interface HookLayoutCallback {
    fun onCreateViewCallback(parent: View?, name: String?, context: Context?, attrs: AttributeSet?, destView: View?)
}